code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -379,6 +379,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Hooch"
 	glass_desc = "You've really hit rock bottom now... your liver packed its bags and left last night."
 
+/datum/reagent/consumable/ethanol/hooch/on_mob_life(mob/living/carbon/M)
+	if(M.mind && M.mind.assigned_role in "Assistant")
```

Comment:
`assigned_role` is a string, `"Assistant"` is a string, string-in-string won't return useful results

Reply:
I don't know why I did that.